composer.json

```diff
@@ -22,7 +22,8 @@
     },
     "require-dev": {
         "symfony/phpunit-bridge": "^4.4 || ^5.0",
-        "nyholm/psr7": "^1.1"
+        "nyholm/psr7": "^1.1",
```

Comment:
let's move this first to have alpha order
also, didn't you want to move http-foundation to reuire-dev?

Don't miss updating the branch-alias and the changelog

Reply:
Alright, I wasn't sure if you wanted me to do that myself. I can move `http/foundation` to dev.